### PR TITLE
[docs] Clarifying RDS binlog requirement

### PR DESCRIPTION
**Pull request by gunnarmorling**
Hey @jpechane quick doc update. Just cost me half a day or so :)

### CODE REVIEW

File: documentation/modules/ROOT/pages/connectors/mysql.adoc [hunk 1]
```diff
@@ -2130,6 +2130,9 @@ mysql> SELECT variable_value as "BINARY LOGGING STATUS (log-bin) ::"
 FROM performance_schema.global_variables WHERE variable_name='log_bin';
 ----
 
+. If you are running MySQL on Amazon RDS, make sure to enable automated backups for your database instance. Otherwise, the binlog will not be enabled, despite of the settings described above.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/mysql.adoc:2133]:
```suggestion
. If you run MySQL on Amazon RDS, you must enable automated backups for your database instance for binary logging to occur. 
If the database instance is not configured to perform automated backups , the binlog is disabled, even if you apply the settings described in the previous steps.
```

File: documentation/modules/ROOT/pages/connectors/mysql.adoc [hunk 2]
```diff
@@ -2130,6 +2130,10 @@ mysql> SELECT variable_value as "BINARY LOGGING STATUS (log-bin) ::"
 FROM performance_schema.global_variables WHERE variable_name='log_bin';
 ----
 
+. If you run MySQL on Amazon RDS, you must enable automated backups for your database instance for binary logging to occur. 
+If the database instance is not configured to perform automated backups , the binlog is disabled, even if you apply the settings described in the previous steps.
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/connectors/mysql.adoc:2134]:
```suggestion
If the database instance is not configured to perform automated backups, the binlog is disabled, even if you apply the settings described in the previous steps.
```

Comment (roldanbob) [reply documentation/modules/ROOT/pages/connectors/mysql.adoc:2134]: There's an extra space before the comma.

**Comment by gunnarmorling** [comment]
Went ahead and merged this one, given it only updates docs no code.

**Comment by jpechane** [comment]
@gunnarmorling Thanks! Sorry for the delay as I was away on Fri.

**Pull request merged**